### PR TITLE
Version bump

### PR DESCRIPTION
**Pull request by qw3ry**
Required changes for the next kernel version

### CODE REVIEW

File: src/test/java/sirius/web/service/JsonOutputSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        given:
+        OutputStream os = new ByteArrayOutputStream();
+        JSONStructuredOutput out = new JSONStructuredOutput(os, null, "UTF8");
+        when:
+        out.beginResult("test");
+        out.beginObject("1", Attribute.set("a", "b"))
+        out.endObject()
+        out.endResult();
+        then:
+        compareJsonString(os.toString(), """{"1":{"a":"b"}}""");
```
Comment (meg1502) [inline src/test/java/sirius/web/service/JsonOutputSpec.groovy:27]:
```suggestion
os.toString() == """{"1":{"a":"b"}}"""
```

shouldn't this be sufficient also ?

**Review by meg1502 — APPROVED** [review]

File: src/test/java/sirius/web/service/JsonOutputSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    private def compareJsonString(String str1, String str2) {
```
Comment (andyHa) [inline src/test/java/sirius/web/service/JsonOutputSpec.groovy:30]: not needed?

**Review by andyHa — CHANGES_REQUESTED** [review]
...

**Review by sabieber — APPROVED** [review]

**Review by bwiedmann — APPROVED** [review]

**Review by mkeckmkeck — APPROVED** [review]

File: src/test/java/sirius/web/service/JsonOutputSpec.groovy (hunk 1 above)
Comment (qw3ry) [reply src/test/java/sirius/web/service/JsonOutputSpec.groovy:27]: It is, indeed. But I there are other correct serializations of this JSON, so the test might break when the JSON is serialized differently. Would it be reasonable to do a simple string equality here?

File: src/test/java/sirius/web/service/JsonOutputSpec.groovy (hunk 2 above)
Comment (qw3ry) [reply src/test/java/sirius/web/service/JsonOutputSpec.groovy:30]: Depends on https://github.com/scireum/sirius-web/pull/747#discussion_r482841806

File: src/test/java/sirius/web/service/JsonOutputSpec.groovy (hunk 1 above)
Comment (meg1502) [reply src/test/java/sirius/web/service/JsonOutputSpec.groovy:27]: I don't really know, i thought we wanted the JSON to always look like this :)

**Pull request merged**